apps/designer/app/routes/rest/project/clone.$domain.ts

```diff
@@ -1,5 +1,7 @@
 import { type LoaderFunction, redirect } from "@remix-run/node";
 import type { User } from "@webstudio-is/prisma-client";
+import { db as database } from "@webstudio-is/project";
+import type { Project } from "@webstudio-is/project";
 import * as db from "~/shared/db";
```

Comment:
these 2 db semantics are problem, one could be projectDb, another could be the userDb